scripts/test.py

```diff
@@ -9,6 +9,9 @@
 WINDOWS = sys.platform.startswith('win')
 MACOS = sys.platform == 'darwin'
 
+assert 'EM_CONFIG' in os.environ, "emsdk should be activated before running this script"
```

Comment:
don't we test activation here? must we do it before as well?

Reply:
(and does an activation before not change the tests for activation in this script?)

Reply:
Activating and sdk is not enough; that only creates the config files.

One also needs to run `source emsdk_env.sh` to activate it within the current shell.

Reply:
Oh, I think the terminology confused me here. `emsdk activate` does stuff like copy precompiled libc, etc. I assumed that the `activate` you meant.

Reply:
It is a little confusing isn't it.   I guess the job of `activate` is to decide which tools are active.. but unless you run `emsdk_env` when the emsdk itself as a whole is not active if your current shell.

Perhaps we should remind users who run `./emsdk activate` in shells without EMSDK that they still have one more step to actually activate in the current shell.